src/epsilon/earley.d

```diff
@@ -347,9 +347,9 @@ void Parse(int Dom, int Affixform, ref int Tree, bool Def) @safe
             Scanner;
             if (CurList == NextItem)
             {
-                import log : error;
+                import epsilon.analyzer : Error;
 
-                error!"%s affixform incorrect"(MSymBuf[CurSym].Pos);
+                Error(MSymBuf[CurSym].Pos, "affixform incorrect");
```

Comment:
Is it just the format that's wrong? Then I would prefer just fixing the format:

    error!"affixform incorrect\n%s"(MSymBuf[CurSym].Pos);

Or do you also need the incremented error count?

Reply:
Yes, just the format is wrong. At least, that's what I have encountered because my regex did not match the original Erarly parser error reporting.
... I was not aware of an incremented error counter! May be it is also needed to be incremented in that case? We should check.

Reply:
Shouldn't the error counter be incremented for Earley parser errors too? Earley parsing is part of the analyzer phase... I would say yes. In that sense my first solution of calling `analyzer.Error()` here would be right!
Additionally, calling that analzer method ensures the right right format for all analyzer errors and establish one and only one point to change the error reporting format. Both points are important for the language server support interface.

Reply:
If `earley` calls `analyzer.Error`, this would cause a cyclic dependency.
Currently, the `analyzer` checks the parse tree returned by `earley.Parse` to increment the error count.
For the moment, this should be enough.